CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master (unreleased)
 
+## 0.8.0
```

Comment:
just leave this as master for now, we'll add 0.8.0 when we cut the release (separate pr)

Reply:
(nmd, i guess this is fine to do all in one pr)